exercises/pascals-triangle/lib/example.dart

```diff
@@ -0,0 +1,24 @@
+class PascalsTriangle {
+  List<List<num>> rows(int rows) {
```

Comment:
Does it have to be generic `num`? Can we not use `int`

Reply:
No, it doesn't have to be `num`, I just think that's how it got generated. I'll make a note to change it.

Reply:
Changed everything to `int`